libgit.el

```diff
@@ -32,6 +32,11 @@
 (unless module-file-suffix
   (error "Module support not detected, libgit can't work"))
 
+(defcustom libgit-cmake-generator nil
+  "CMake generator to use."
```

Comment:
Someone who is not familiar with cmake (like me), should still be able to understand what this is all about. As I understand it, this creates build instructions that can later be used by `cmake --build` instead of creating a Makefile later to be used by `make`.